src/redux/books/books.js

```diff
@@ -0,0 +1,21 @@
+// action types
+const ADD_BOOK = 'books/books/ADD_BOOK';
+const REMOVE_BOOK = 'books/books/REMOVE_BOOK';
+
+// reducer
+const initialState = [];
+
+export default function reducer(state = initialState, action) {
+  switch (action.type) {
+    case ADD_BOOK:
+      return state;
+    case REMOVE_BOOK:
+      return state;
+    default:
+      return state;
+  }
+}
```

Comment:
- [ ] As stated the project requirement after your configure your Redux Store and write actions and reducer for adding and removing books, Kindly make sure that adding and removing actions are implemented. Because When I try to add new book nothing happens this because of
- When  action.type is ADD_BOOK  I recommend you to use the function that returns a new state object in which the books array will contain a new book object, passed by action.payload. But in the case of yours, you return an state with an empty array.Just like this 
```
 return state.concat({
        ...action.payload,
      });
```

- When action.type is REMOVE_BOOK I recommend you to use the function using ES6 filter() method to create a new array, which will not contain the book you want to remove from the store (filter by the id key ) - i.e.:
`return state.filter(book => book.id !== id`

Reply:
Thanks, included. However,  the functionalities you pointed out are in the next project's requirements

@Abenezer-Tilahun